src/edu/cornell/mannlib/vitro/webapp/controller/individual/IndividualResponseBuilder.java

```diff
@@ -87,6 +89,25 @@ ResponseValues assembleResponse() throws TemplateModelException {
 		if (dcoId != null) {	
 			body.put("dcoId", dcoId.getRdfsLabel());
 		}
+		/*
+		 * Start of Modification for networkId
+		 */
+		String networkId = individual.getDataValue("http://info.deepcarbon.net/schema#networkId");
```

Comment:
I don't understand how this is going to work if we're grabbing the wrong property? For my instaince data I don't not have a property dcoOntoNamespace+"networkId". Does this mean we have to change all of our instance data, the field we use to link together a profile and an account, the user registration code to push the right value?